website/cue/reference/components/transforms/base/exclusive_route.cue

```diff
@@ -0,0 +1,29 @@
+package metadata
```

Comment:
Note that, in addition to the cue doc you mentioned is still TODO, you'll need to create a markdown file for the transform too.

Reply:
> create a markdown file for the transform too.

Updated the PR with a lot of doc related changes. Not sure about this comment. Let me know if I missed something 🙏

Reply:
I think you are still missing the markdown file so the new transform docs won't be rendered on the website. You'll need to create `website/content/en/docs/reference/configuration/transforms/exclusive_route.md`. Hugo uses that to generate the transform page (theoretically these pages could be generated from the cue docs, I think, but they currently aren't).

Reply:
I think this new markdown file is still missing.